diff --git a/Makefile b/Makefile
index aae16e6..5972a3e 100644
--- a/Makefile
+++ b/Makefile
@@ -4,11 +4,11 @@
 ###
 
 fbgrab: fbgrab.c
-	gcc -g -Wall fbgrab.c -lpng -lz -o fbgrab
+	$(CC) -Wall ${CPPFLAGS} ${CFLAGS} ${LDFLAGS} fbgrab.c -lpng -lz -o fbgrab
 
 install:
-	install fbgrab /usr/bin/fbgrab
-	install fbgrab.1.man /usr/man/man1/fbgrab.1
+	install fbgrab $(prefix)/bin/fbgrab
+	install fbgrab.1.man $(prefix)/share/man/man1/fbgrab.1
 
 clean:
 	rm -f fbgrab *~ \#*\#
